<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="DropShadow">
      <property name="styleSheet">
       <string notr="true">QFrame{
	color: rgb(253, 253, 253);
	background-color: rgb(253, 253, 253);
	border-radius: 08 px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Title">
       <property name="geometry">
        <rect>
         <x>204</x>
         <y>40</y>
         <width>213</width>
         <height>77</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>LEIXO</family>
         <pointsize>42</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(112, 112, 112);
}</string>
       </property>
       <property name="text">
        <string>ELIXIR</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="LoadingText">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>282</y>
         <width>620</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(112, 112, 112);
}</string>
       </property>
       <property name="text">
        <string>Loading</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="ProgressBar">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>318</y>
         <width>500</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Roboto</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar{
	background-color: rgb(222, 222, 222);
	color: rgb(165, 165, 165);
	border-style: none;
	border-radius: 2px;
}

QProgressBar::chunk{
	background-color: rgb(112, 112, 112);
	border-radius:2px;
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
